pip/wheel.py

```diff
@@ -591,6 +591,8 @@ def __init__(self, requirement_set, finder, build_options=None,
         self.requirement_set = requirement_set
         self.finder = finder
         self._cache_root = requirement_set._wheel_cache._cache_dir
+        self._ephem_cache_root = (
+            requirement_set._wheel_cache._ephem_cache_dir.path)
```

Comment:
This looks pretty bad. 😕

I guess this can stay for now because I would be moving caches around as a part of my GSoC project, so, all this would likely change that time anyway.